libs/community/langchain_community/tools/azure_cognitive_services/content_safety.py

```diff
@@ -0,0 +1,151 @@
+from __future__ import annotations
+
+import logging
+import os
+from typing import Any, Dict, Optional
+
+from langchain_core.callbacks import CallbackManagerForToolRun
+from langchain_core.tools import BaseTool
+
+logger = logging.getLogger(__name__)
+
+
+class AzureContentSafetyTextTool(BaseTool):
+    """
+    A tool that interacts with the Azure AI Content Safety API.
+
+    This tool queries the Azure AI Content Safety API to analyze text for harmful
+    content and identify sentiment. It requires an API key and endpoint,
```

Comment:
While I agree that in some ways we are identifying the sentiment of the text with content safety, this might be confused with the NLP sentiment analysis that Azure AI textanalytics service performs. I think we should stick to presenting this as a content safety tool which can detect harmful user-generated and AI-generated content